test/integration/framework/flags.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"flag"
+
+	"k8s.io/component-base/logs"
+)
+
+func init() {
+	// All integration tests should have the same logging flags as other
+	// commands.
+	logs.AddGoFlags(flag.CommandLine)
```

Comment:
I should import `component-base/logs/init` here instead of having a custom init function. The advantage is that the import can appear multiple times in a binary and will add flags only once, which is important because `flag.CommandLine` may panic when flags get redefined.